tests/unit/common/test_encryption.py

```diff
@@ -89,14 +89,17 @@ def test_gen_key(self, temp_output_dir: str):
         assert key_gen_contents != ""
 
     def test_get_key(
-        self, merlin_server_dir: str, test_encryption_key: bytes, redis_results_backend_config: "fixture"  # noqa: F821
+        self,
```

Comment:
one more tab?

Reply:
I think the tabs here are correctly aligned since the tests are held within a class. Our linter likely made this change when I ran `make fix-style`

Reply:
style-wars